src/components/NewClusterModal.js

```diff
@@ -182,6 +182,11 @@ export const NewClusterModal = withModalDrawer({ width: 675 })(class NewClusterM
       } else {
         this.setState({ isCustomEnv: true, customEnvImage: dockerImage })
       }
+
+      const { jupyterUserScriptUri } = await Ajax().Jupyter.cluster(currentCluster.googleProject, currentCluster.clusterName).details()
```

Comment:
Since we're already making this call above, we could just pull this additional information out above instead of making the call again.